packages/block-editor/src/components/inserter/style.scss

```diff
@@ -268,6 +268,10 @@ $block-inserter-tabs-height: 44px;
 
 .block-editor-inserter__quick-inserter {
 	width: $block-inserter-width;
+
+	@media (max-width: #{ ($break-medium) }) {
+		width: 100%;
+	}
```

Comment:
Can we switch this (mobile-first) and use the break-medium mixin like we do in other places?

Reply:
Pushed a commit to address this.